zcash_client_sqlite/src/wallet/init/migrations.rs

```diff
@@ -79,12 +80,12 @@ pub(super) fn all_migrations<P: consensus::Parameters + 'static>(
     //            \                          \     ephemeral_addresses     /                       /
     //             \                          \            |              /                       /
     //              ------------------------------ tx_retrieval_queue ----------------------------
-    //                                                     |
-    //                                            support_legacy_sqlite
-    //                                                     |
-    //                                         fix_broken_commitment_trees
-    //                                                     |
-    //                                          fix_bad_change_flagging
+    //                                               /           \
+    //                            support_legacy_sqlite         add_account_uuids
```

Comment:
Is there any need for this to be parallel with the branch up to `fix_bad_change_flagging`? I thought we decided that we would only use parallel migrations when we actually had a need for it, to simplify reasoning about the possible database states.

Reply:
It is explicitly required because the test for `fix_bad_change_flagging` creates an account, and the code now requires a `uuid` column. Preserving the old account creation code inside the test doesn't work in this case because it is indirectly used through the test harness. If we made `add_account_uuids` be a child of `fix_bad_change_flagging` then we could not run the test, because the test needs to initialise its state without `fix_bad_change_flagging` applied (which would then mean `add_account_uuids` is also not applied).

So yes, I would say that we have an actual need for parallel migrations here. (And also, we now have tracking of the leaf states as of various releases, so the possible database states are much easier to reason about now than they were when the recent global merge happened.)